apps/addresses/models.py

```diff
@@ -120,7 +123,7 @@ def get_static_google_map_image_url(self):
         height = 234
         str_value = self.__str__()
         base_url = 'http://maps.googleapis.com/maps/api/staticmap?'
-        query = urllib.urlencode(
```

Comment:
## REVIEWER NOTE:
`urllib` from `six.moves` has no `urlencode` instead it should be imported from `urllib.parse`